hazelcast/src/main/java/com/hazelcast/agrona/collections/Int2ObjectHashMap.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2014 Real Logic Ltd.
```

Comment:
Licensing issue always confuse me. But I think you are right we need a NOTICE file . I saw this example in an apache project:
https://github.com/apache/karaf/blob/master/NOTICE